app/models/exam.rb

```diff
@@ -0,0 +1,4 @@
+class Exam < ApplicationRecord
+  belongs_to :student
+  belongs_to :tutor
```

Comment:
You need to reflect these in the tutor and student models as well.
`has_many :exams` in each one, if I'm not forgetting my syntax.